packages/scss/src/components/_container.scss

```diff
@@ -26,34 +26,34 @@
 
 .container {
 	&.mod-sm {
-		max-width: $sm;
+		max-width: _get($luTheme, "breakpoints.sm");
 	}
 
 	&.mod-md {
-		max-width: $md;
+		max-width: _get($luTheme, "breakpoints.md");
 	}
 
 	&.mod-lg {
-		max-width: $lg;
+		max-width: _get($luTheme, "breakpoints.lg");
 	}
 
 	&.mod-xl {
-		max-width: $xl;
+		max-width: _get($luTheme, "breakpoints.xl");
 	}
 
 	&.mod-navSide {
-		width: $xl - 170px;
+		width: _get($luTheme, "breakpoints.xl") - 170px;
```

Comment:
As it's a value relative to the nav-side component (the aside main menu), it should be in a variable. 
Do you think that the nav-side component width should be also overridable?